config/config.go

```diff
@@ -11,13 +14,25 @@ var (
 	SDStoreURL string
 	// SDToken is SD_TOKEN value
 	SDToken string
+	// SDArtifactsDir is SD_ARTIFACTS_DIR value
+	SDArtifactsDir string
+	// BaseCommandPath is path of installing binary command
+	BaseCommandPath = "/opt/sd/commands/"
 )
 
+func addSlash(val string) string {
+	if val != "" && val[len(val)-1] != '/' {
```

Comment:
This function just add `/` function. Not checking the value of env, so I think this function do not need to return error. 

Of course it maybe cause error if there are blank env values, but I think it will be handled by other error check places.

Or I will create new function it check the env values which is blank or not.

Reply:
Is this append slash normalization necessary?
We should not modify environmental variables.

Reply:
When I get the url from `SD_API_URL`, there is possibility `/api/url` or `/api/url/`.  
If we make the url from the `SD_API_URL` it will be annoying to check `/`, so I create this function.

Reply:
How about using `path.Join()` after its URL is parsed by the `net/url`?
https://golang.org/pkg/net/url/

Reply:
That's more like it. 
I will fix.